release.csx

```diff
@@ -0,0 +1,84 @@
+#r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
+#r ".\packages\system.runtime.interopservices.runtimeinformation\4.0.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
+
+using Octokit;
+
+const string repoOwner = "FakeItEasy";
+const string repoName = "FakeItEasy";
+const string remote = "upstream";
+
+var version = Args.FirstOrDefault();
+if (version == null)
+{
+    throw new Exception("No version supplied");
+}
+
+var releaseBranchName = $"release/{version}";
+
+var token = ReadTokenFromNetrc();
+
+var gitHubClient = new GitHubClient(new ProductHeaderValue("FakeItEasy-build-scripts"));
+gitHubClient.Credentials = new Credentials(token);
+Cmd("git", $"checkout -b {releaseBranchName}");
+Cmd("git", $"push {remote} {releaseBranchName}");
+
+Console.WriteLine($"Creating pull request to merge {releaseBranchName} into master...");
+var pr = await gitHubClient.PullRequest.Create(
+    repoOwner,
+    repoName,
+    new NewPullRequest($"Release {version}", releaseBranchName, "master"));
+Console.WriteLine($"Created pull request {pr.Number}: '{pr.Title}'");

```

Comment:
Assuming a merge into master, and we're not (yet?) making a PR to go into develop. (I guess in that case, we could git checkout develop.)
So we're not supporting release of hotfixes. Just "regular" releases.